test/etools-datepicker_test.html

```diff
@@ -20,10 +21,27 @@
 
     <script>
       suite('etools-datepicker', function() {
+        var basic
+        setup(function() {
+          basic = fixture('basic')
+        });
         test('instantiating the element works', function() {
-          var element = fixture('basic');
-          assert.equal(element.is, 'etools-datepicker');
+          assert.equal(basic.is, 'etools-datepicker');
+        });
+        test('tapping datepicker icon toggles it open', function() {
+          expect(dialog.attributes[4].name).to.equal('aria-hidden')
+          var calendarIcon = Polymer.dom(basic.root).querySelector('iron-icon')
+          MockInteractions.tap(calendarIcon)
+          expect(dialog.attributes[4].name).to.not.equal('aria-hidden')
         });
+        test('tapping a date selects it', function() {
+          var calendarIcon = Polymer.dom(basic.root).querySelector('iron-icon')
+          MockInteractions.tap(calendarIcon)
+          var firstDay = dialog.children.picker.querySelector('.day-item.selection')
+          var firstDayValue = firstDay.attributes.date.value
+          MockInteractions.tap(firstDay)
+          expect((new Date(firstDayValue + 'T00:00:00' + '-04:00')).toLocaleDateString()).to.equal(picker.date.toLocaleDateString())
```

Comment:
Use UTC Timezone, do not hardcode timezone offset ... if you run this in other timezone the test will fail.

Reply:
good catch; i'll change this.